src/services/cssNavigation.ts

```diff
@@ -422,7 +422,22 @@ export class CSSNavigation {
 			return this.mapReference(await this.resolveModuleReference(target, documentUri, documentContext), isRawLink);
 		}
 
-		const ref = await this.mapReference(documentContext.resolveReference(target, documentUri), isRawLink);
+		// Treat bare module names (“bootstrap/...”) like sass-loader does
+		const startsWithSchemeRegex = /^\w[\w\d+.-]:/;
```

Comment:
Node module resolving should only happen when `this.resolveModuleReferences` is true.
(only set for LESS and SCSS).

So this needs to go behind the `if (this.resolveModuleReferences) {`.
I looks like we already call `resolveModuleReference` there, so this code is not necessary?